packages/trip-details/i18n/en-US.yml

```diff
@@ -0,0 +1,12 @@
+# Default messages for the TripDetails component.
+# To use from a react-intl application, import the content of this file into the messages object
+# that has your other localized strings, and that you pass to the IntlProvider component.
+otpUi.TripDetails.calories: "Calories Burned: <b>{calories, number, ::.}</b>"
```

Comment:
I think it'd look a lot nicer to write out the hierarchy of these and then flatten in the code.

Reply:
Updated in https://github.com/opentripplanner/otp-ui/pull/281/commits/21beebcf571549e35ac87db6072146d47d737417, ended up looking better that way!